bin/resources/GameIndex.yaml

```diff
@@ -64136,6 +64136,11 @@ SLUS-21113:
   name: "Atelier Iris - Eternal Mana"
   region: "NTSC-U"
   compat: 5
+  gameFixes:
+    - SoftwareRendererFMVHack # Fixes horizontal lines in FMV and prevents hash cache from disabling itself
+  gsHWFixes:
+    roundSprite: 2 # Fixes character portraits when upscaling and reduces lines in FMVs when using HW renderer
+    texturePreloading: 2 # Fixes flickering black lines around sprites when upscaling.
```

Comment:
This isn't needed.

Reply:
Saw the failure and just changed it to 1 about 30 seconds before I saw this comment.  Having the option enabled does fix some errant black lines (example above), although having it set to 1 or 2 (`Partial` or `Full`) both seem to fix the issue the same way.

Is it better to keep it as 1 and fix the issue (which would also match other Gust games in the GameIndex)? Or just remove the option completely and let the end-user mess with it?

Reply:
We warn for anything under Full so it wouldn't be needed here.